src/popup/router/pages/Send.vue

```diff
@@ -253,15 +257,17 @@ export default {
           const { hash } = await this.$store.dispatch('fungibleTokens/transfer', [
             receiver,
             this.form.amount,
-            { waitMined: false, modal: false },
+            { waitMined: true, modal: false },
```

Comment:
Are you sure want use to wait for the transaction? This is inconsistent with a usual sending

Reply:
Yes, I want to wait for it for FT, because currently we are using the list of available tokens per account from the backend and the cache is not fast enough. Which fires additional errors if not awaited.

Once MDW has the capability to track that for us - we will remove it.